lib/iris/tests/test_cell.py

```diff
@@ -96,6 +96,9 @@ def test_in(self):
         assert c not in [3, c2]
 
     def test_coord_equality(self):
+        # NOTE: These comparisons intentionally use explicit operators to ensure each
+        # rich comparison method (__eq__, __ne__, __lt__, __le__, __gt__, __ge__) is 
```

Comment:
As you may have noticed, our linter picked up a tiny problem. Here is the fix:

```suggestion
        # NOTE: These comparisons intentionally use explicit operators to ensure each
        # rich comparison method (__eq__, __ne__, __lt__, __le__, __gt__, __ge__) is
```